doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -350,6 +350,35 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-5-0 v2-5-1 v2-5-2 v2-5-3 v2-6-0 v2-6-1
+
+KRKNWK-18916: If the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` is selected there are two issues:
```

Comment:
```suggestion
KRKNWK-18916: If the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` is selected there are two issues:
```
```suggestion
KRKNWK-18916: Issues related to the  :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` Kconfig option
  When the Kconfig option is selected, there are two issues:
```
The formatting requires a description line without an empty line, which can't be done with a bullet point.